torch/_refs/nn/functional/__init__.py

```diff
@@ -297,6 +299,38 @@ def _check_reduction_value(reduction: str):
         raise ValueError("{} is not a valid value for reduction".format(reduction))
 
 
+def _get_string_reduction_arg(
+    size_average: Optional[bool], reduce: Optional[bool]
```

Comment:
I would just make these both required kwarg-only arguments for now

Also add a comment describing the helper